Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md

```diff
@@ -0,0 +1,67 @@
+---
+tags: engineering/backend, nghenhan, hedge-foundation
+author: Minh Tran
+github_id: thminhVN
+date: 2023-05-07
+---
+
+# Introduction
```

Comment:
Should down level all headings to 1 level

Reply:
updated